python_modules/libraries/dagster-dbt/dagster_dbt_tests/components/test_dbt_project_component.py

```diff
@@ -127,15 +127,14 @@ def get_additional_scope(cls) -> Mapping[str, Any]:
         ),
         ({"tags": {"foo": "bar"}}, lambda asset_spec: asset_spec.tags.get("foo") == "bar", False),
         (
-            {"kinds": ["snowflake"]},
-            lambda asset_spec: "snowflake" in asset_spec.kinds
-            and "dbt" in asset_spec.kinds,  # Ensure dbt-specific kind is not overwritten
+            {"kinds": ["snowflake", "dbt"]},
```

Comment:
why is this necessary? i think i get the picture (we used to get_asset_spec before a separate post-processing step in dagster-dbt) but is there a way to make this change w/ updating this behavior?

Reply:
Ya the `dbt` insertion was just a consequence of how things were called before. 

I was trying to stay in line with the existing two behaviors of `merge` vs `replace` and I was less certain about promoting `kinds` to `merge` since its currently `replace`. 

I checked with @benpankow and this seemed reasonable to him.

Reply:
> I was trying to stay in line with the existing two behaviors of merge vs replace and I was less certain about promoting kinds to merge since its currently replace.

imo promoting `kinds` to merge makes sense to me but doesn't have to be this PR